Source/Csla/Serialization/SerializationFormatterFactory.cs

```diff
@@ -19,7 +19,9 @@ public static class SerializationFormatterFactory
     /// </summary>
     public static ISerializationFormatter GetFormatter(ApplicationContext applicationContext)
     {
-      return (ISerializationFormatter)applicationContext.CreateInstanceDI(ApplicationContext.SerializationFormatter);
+      var options = applicationContext.GetRequiredService<Configuration.CslaOptions>();
```

Comment:
This whole method/type can be removed if we move to a pure DI "based" config.

Reply:
Removed formatter factory